cfgov/v1/migrations/0024_create_referred_feedback_model_20161110_1839.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
```

Comment:
Minor comment: since this migration filename has the index (`0024`) and the custom name, the date and timestamp isn't needed.